blues/moves.py

```diff
@@ -146,17 +146,21 @@ class RandomLigandRotationMove(Move):
     ligand.positions : np.array of ligands positions
     """
 
-    def __init__(self, structure, resname='LIG'):
+    def __init__(self, structure, resname='LIG', theta_cutoff=90.0):
         """Initialize the model.
         Parameters
         ----------
         resname : str
             String specifying the resiue name of the ligand.
         structure: parmed.Structure
             ParmEd Structure object of the relevant system to be moved.
+        theta_cutoff : float
+            Cutoff in degrees for random rotational matrix. Values smaller
+            than cutoff will result in generating a new rotation matrix.
         """
 
         self.resname = resname
+        self.theta_cutoff = theta_cutoff
```

Comment:
Maybe we should have a sanity check for the theta_cutoff. Since we're doing a while loop using the cutoff, we could be stuck in an infinite loop if `theta_cutoff >= 180`, so throwing a `ValueError` if it's >=180 would be good.